app/questionPage.html

```diff
@@ -89,7 +128,9 @@ <h4 class="timer"></h4>
 <div id="overview_page">
     <div class="container">
         <div class="row">
-            <img class="logo" alt="iO Academy Logo" src="./images/logos/SVGs/iO_Logo_Alt.svg">
+            <div class="col-6">
+                <img class="logo" alt="iO Academy Logo" src="./images/logos/SVGs/iO_Logo_Alt.svg">
+            </div>
```

Comment:
what difference does this make? i don't see any difference with or without the wrapping div

Reply:
This is removed